### ISSUE TITLE
zsh support

### ISSUE THREAD

**Issue by spenoir** [fabiospampinato/vscode-terminals#25]
I have zsh installed and vscode terminal uses zsh with the config:

`"terminal.integrated.shell.osx": "/bin/zsh",`

When I open my project with the extension enabled and a terminals.json file I get the following error:

`The terminal process command '/bin/zsh' failed to launch (exit code: 1)`

**Comment by fabiospampinato** [comment]
I don't think this has anything to do with zsh, since I'm using it too, unless you have configured it badly or something.

Could you paste your `terminals.json` configuration?

**Comment by spenoir** [comment]
What setting do you have for: `terminal.integrated.shell`?

my terminals.json

```
{
    "autorun": true,
    "autokill": true,
    "env": {
        "name": "value"
    },
    "terminals": [
        {
            "name": "Foo",
            "description": "Bar",
            "icon": "code",
            "cwd": "/Users/me/project",
            "commands": [
                "./run.sh",
                "docker-compose up"
            ]
        }
    ]
}    
```

**Comment by fabiospampinato** [comment]
> What setting do you have for: terminal.integrated.shell?

I have:

```json
"terminal.integrated.shell.osx": "/bin/zsh",
```

Are you sure the `cwd` path is right? because I can only reproduce this if that points to a non existent directory.

**Comment by spenoir** [comment]
You're right, some of my `cwd` paths were wrong, it works perfectly now. Thanks for your help

**Comment by fabiospampinato** [comment]
Maybe we should detect this sort of errors in the configuration 🤔

**Comment by fabiospampinato** [comment]
> Maybe we should detect this sort of errors in the configuration 🤔

Implemented in v1.12.5 👍
